### PR TITLE
changelog for 1.8.12

### PR DESCRIPTION
**Pull request by sryza**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,4 +1,39 @@
-# Changelog
+# Changelog 
+
+## 1.8.12 (core) / 0.24.12 (libraries)
+
+### New
+
+* The `AutomationCondition.eager()`, `AutomationCondition.missing()`, and `AutomationCondition.on_cron` conditions are now compatible with asset checks. [#25170](https://github.com/dagster-io/dagster/pull/25170)
```
Comment (OwenKephart) [inline CHANGES.md:7]: Historically, we've removed the links to the specific PRs

File: CHANGES.md [hunk 2]
```diff
@@ -1,4 +1,39 @@
...
+* Added `AssetSelection.materializable()``, which returns only assets that are materializable in an existing selection [#25076](https://github.com/dagster-io/dagster/pull/25076)
```
Comment (OwenKephart) [inline CHANGES.md:8]: double backtick here

File: CHANGES.md [hunk 3]
```diff
@@ -1,4 +1,39 @@
...
+* Added a new AutomationCondition.all_deps_blocking_checks_passed condition, which can be used to prevent materialization when any upstream blocking checks have failed. [#25171](https://github.com/dagster-io/dagster/pull/25171)
```
Comment (OwenKephart) [inline CHANGES.md:9]: backticks here

File: CHANGES.md [hunk 4]
```diff
@@ -1,4 +1,39 @@
...
+* Added a `code_version` parameter to the `@graph_asset` decorator [#25219](https://github.com/dagster-io/dagster/pull/25219)
+* If a `LaunchPartitionBackfill` mutation is submitted to GQL with invalid partition keys, it will return an early `PartitionKeysNotFoundError`. [#25107](https://github.com/dagster-io/dagster/pull/25107)
```
Comment (OwenKephart) [inline CHANGES.md:11]: it will return -> it will now return

**Review by shalabhc — APPROVED** [review]

File: CHANGES.md [hunk 5]
```diff
@@ -1,4 +1,39 @@
...
+
+### Documentation
+
+* [dagster-k8s] `DagsterK8sPipesClient.run` now shows up in API docs. [#24678](https://github.com/dagster-io/dagster/pull/24678)
+
+### Dagster Plus
+
+* [ui] Fixed a bug in the catalog UI where owners filters were not applied correctly. [#25142](https://github.com/dagster-io/dagster/pull/25142)
+* [ui] Fixed width of the column lineage dropdown selector on the asset page. [#12024](https://github.com/dagster-io/internal/pull/12024)
+* [ui] Column lineage now correctly renders when set on asset definition metadata [#12110](https://github.com/dagster-io/internal/pull/12110)
```
Comment (OwenKephart) [inline CHANGES.md:35]: end with period

File: CHANGES.md [hunk 6]
```diff
@@ -1,4 +1,39 @@
...
+* [dagster-dbt] Fixed an issue where column metadata generated with `fetch_column_metadata` did not work properly for models imported through dbt dependencies [#25185](https://github.com/dagster-io/dagster/pull/25185)
+
+### Documentation
+
+* [dagster-k8s] `DagsterK8sPipesClient.run` now shows up in API docs. [#24678](https://github.com/dagster-io/dagster/pull/24678)
+
+### Dagster Plus
+
+* [ui] Fixed a bug in the catalog UI where owners filters were not applied correctly. [#25142](https://github.com/dagster-io/dagster/pull/25142)
+* [ui] Fixed width of the column lineage dropdown selector on the asset page. [#12024](https://github.com/dagster-io/internal/pull/12024)
```
Comment (shalabhc) [inline CHANGES.md:34]: I dont think we want to link internal pull requests?

**Review by OwenKephart — CHANGES_REQUESTED** [review]

**Review by shalabhc — CHANGES_REQUESTED** [review]
links to internal pull requests should be removed

File: CHANGES.md (hunk 6 above)
Comment (sryza) [reply CHANGES.md]: Good point

File: CHANGES.md (hunk 1 above)
Comment (sryza) [reply CHANGES.md]: Took these out. FWIW I think it would be useful to include these in the future.

**Review by OwenKephart — APPROVED** [review]

**Pull request merged**